galaxy/tools/deps/requirements.py

```diff
@@ -196,7 +196,7 @@ def __str__(self):
 def parse_requirements_from_dict(root_dict):
     requirements = root_dict.get("requirements", [])
     containers = root_dict.get("containers", [])
-    return ToolRequirements.from_list(requirements), map(ContainerDescription.from_dict, containers)
+    return ToolRequirements.from_list(requirements), [ContainerDescription.from_dict(c) for c in containers]
```

Comment:
You can also use `list(map(...))`, if you prefer the functional syntax, as long as the function is not a lambda.

Reply:
Do you have preference?

Reply:
Not really, but I try to minimise the changes.

Reply:
This feels more natural to me - so I'm going to stick with this - thanks for the heads up though.